social-network/src/your-articles.js

```diff
@@ -0,0 +1,109 @@
+                        "image": "https://cdn-images-1.medium.com/max/800/1*KHs6OgOb7_aJhmQvOKj_dg.jpeg",
+                        "link": "https://electricliterature.com/how-to-write-elevator-pitch-novel-publicity-infographic-a8ec74ecf7ce",
+                        "author": {
+                            "name": "Electric Literature",
+                            "image": "https://cdn-images-1.medium.com/fit/c/80/80/1*bs47qwmiIR_bU01Jm6ZI9Q.jpeg",
+                            "isMediumMember": false
+                        },
+                        "postedDate": "2018-03-28T07:00:00.000Z",
+                        "minutesToRead": 2,
+                        "hasAudioAvailable": false,
+                        "memberPreview": false
+                    },
+                    {
+                        "title": "Do Something You Can't Win At",
+                        "description": "And make it something you love",
+                        "image": "https://cdn-images-1.medium.com/max/800/1*cioT9CBBV2s0svet9LqKZA.png",
+                        "link": "https://medium.com/@krisgage/do-something-you-cant-win-at-cb2e5d2ad3c1",
+                        "author": {
+                            "name": "Kris Gage",
+                            "image": "https://cdn-images-1.medium.com/fit/c/80/80/1*e7o4kbk8ofT6eKaLrO0jaw.png",
+                            "isMediumMember": false
+                        },
+                        "postedDate": "2018-03-30T07:00:00.000Z",
+                        "minutesToRead": 3,
+                        "hasAudioAvailable": false,
+                        "memberPreview": false
+                    },
+                    {
+                        "title": "Art as a Second Language",
+                        "description": "Drawing is a translation of all my voices and words — an attempt to draw nothing",
+                        "image": "https://cdn-images-1.medium.com/max/800/1*_NTwW2r8Konom4d5vIPkuw.png",
+                        "link": "https://medium.com/@bluebed/speaking-different-languages-through-art-710e0a992dd3",
+                        "author": {
+                            "name": "Roman Muradov",
+                            "image": "https://cdn-images-1.medium.com/fit/c/80/80/0*3PeNTwOh_hQpAuw0.png",
+                            "isMediumMember": false
+                        },
+                        "postedDate": "2018-03-27T07:00:00.000Z",
+                        "minutesToRead": 6,
+                        "hasAudioAvailable": true,
+                        "memberPreview": true
+                    }
+                ]
+        }
+    }
+    render() {
+        return (
+             <div >
+                {this.state.data.map((title, i) => <InnerContent key={i}
+                    data={title} />)}
+             </div>
+         );
+    }
+}           
+class InnerContent extends Component {
```

Comment:
Nice job defining this component. Since the other `InnerContent` component is the same, this is an excellent example where you can separate this component into a separate file, then import/use it in both article components.